src/app/beer_garden/db/mongo/models.py

```diff
@@ -585,24 +585,19 @@ class Principal(MongoModel, Document):
     }
 
 
-class RefreshToken(Document):
-    brewtils_model = brewtils.models.RefreshToken
+class UserToken(Document):
+    issued_at = DateTimeField(required=True, default=datetime.datetime.utcnow)
+    expires_at = DateTimeField(required=True)
+    user = LazyReferenceField("User", required=True, reverse_delelete_rule=CASCADE)
+    uuid = UUIDField(binary=False, required=True, unique="True")
 
-    issued = DateTimeField(required=True)
-    expires = DateTimeField(required=True)
-    payload = DictField(required=True)
-
-    meta = {"indexes": [{"fields": ["expires"], "expireAfterSeconds": 0}]}
-
-    def get_principal(self):
-        principal_id = self.payload.get("sub")
-        if not principal_id:
-            return None
-
-        try:
-            return Principal.objects.get(id=principal_id)
-        except DoesNotExist:
-            return None
+    meta = {
+        "indexes": [
+            {"fields": ["expires_at"], "expireAfterSeconds": 0},
+            {"fields": ["user"]},
+            {"fields": ["uuid"]},
+        ]
+    }
```

Comment:
Since I'm not defining any options on the `user` and `uuid` indexes you are correct and I could just define it the way you have it here.  The definition I have is correct, but unnecessarily verbose given that I'm in fact not using any options.  I'll switch to this style since it reads slightly cleaner.